webapp/src/components/Meeting.tsx

```diff
@@ -28,9 +29,11 @@ const Meeting = ({ meetingId }: MeetingProps) => {
   )?.id;
   return (
     <Grid>
-      <h1>{`Meeting on ${formatDate(meeting.starts_at)} at ${formatTime(
-        meeting.starts_at
-      )}`}</h1>
+      {!isInDrawer ? (
+        <h1>{`Meeting on ${formatDate(meeting.starts_at)} at ${formatTime(
```

Comment:
Perhaps this heading should be in the `MeetingPage` component, and not the `Meeting` component. Then this prop wouldn't be necessary.

Reply:
Yeah, I think this would be a good idea.  The meetings page does not fetch the meting info.  We could have a state variable on the declared on the `MeetingPage` component whose update function would be called from the `Meeting` component.  Would that be a good solution?

Reply:
Sounds like we might want to consider moving to a "container components" architecture, where state is managed separately from presentation. Let's talk about this with the team.

Reply:
you mean like redux?

Reply:
Per our discussion, the conditional render of the heading (based on *where* the Meeting component lives) has been removed (screenshot updated) to avoid React anti-patterns.